pkg/utils/utils.go

```diff
@@ -270,6 +319,12 @@ func NeedUpdate(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetworkv1.Int
 							glog.V(2).Infof("NeedUpdate(): VF %d MTU needs update, desired=%d, current=%d", vf.VfID, group.Mtu, vf.Mtu)
 							return true
 						}
+
+						// this is needed to be sure the admin mac address is configured as expected
```

Comment:
why can the admin mac be "externally configured" ? that should be part of sriov configuration performed by the external service.